include/grpc++/impl/codegen/async_stream.h

```diff
@@ -35,6 +35,9 @@ class ClientAsyncStreamingInterface {
  public:
   virtual ~ClientAsyncStreamingInterface() {}
 
+  /// Start the call that was set up by the constructor
+  virtual void StartCall(void* tag) = 0;
```

Comment:
We will need a big warning comment here saying this is only needed when a prepare call was used to create the reader object. I think it might be helpful to buffer the start bit in the object and assert fail if it is true.

Reply:
Excellent suggestion, I did this.